pkg/models/interval.go

```diff
@@ -20,39 +20,32 @@ import (
 
 // Interval a period of time
 type Interval struct {
-	ID        string `json:"id"`
-	Created   int64  `json:"created"`
-	Modified  int64  `json:"modified"`
-	Origin    int64  `json:"origin"`    // not currently used reserved for interval origin source
-	Name      string `json:"name"`      // non-database identifier for a shcedule (*must be quitue)
-	Start     string `json:"start"`     // Start time i ISO 8601 format YYYYMMDD'T'HHmmss 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyymmdd'T'HHmmss")
-	End       string `json:"end"`       // Start time i ISO 8601 format YYYYMMDD'T'HHmmss 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyymmdd'T'HHmmss")
-	Frequency string `json:"frequency"` // how frequently should the event occur according ISO 8601
-	Cron      string `json:"cron"`      // cron styled regular expression indicating how often the action under interval should occur.  Use either runOnce, frequency or cron and not all.
-	RunOnce   bool   `json:"runOnce"`   // boolean indicating that this interval runs one time - at the time indicated by the start
+	BaseObject
```

Comment:
Not sure we should be adding BaseObject back in here. We want to get rid of it, as discussed, so I believe @xmlviking created this type with inline properties for that reason.

Reply:
Corrected.